colossalai/builder/pipeline.py

```diff
@@ -262,4 +268,4 @@ def build_pipeline_model(layers: nn.Sequential, num_chunks: int = 1, verbose: bo
                 log_str += f'===== chunk={chunk}, layer=[{start}-{end}] =====\n'
                 log_str += '\n'.join([str(layer) for layer in layers[start:end]]) + '\n'
             logger.info(log_str, ranks=[0])
-    return nn.ModuleList(module_list) if len(module_list) > 1 else module_list[0]
+    return nn.Sequential(*tuple(module_list)) if len(module_list) > 1 else module_list[0]
```

Comment:
`ModuleList` is used for interleaved pipeline. Each module in this `ModuleList` is a `Sequential` module. Why do you use `Sequential` here? Actually I think they shouldn't be sequential.

Reply:
I forget to consider the interleaved pipeline. Seems `ModuleList` is fine, I'll test it.

Reply:
I reconsider it. I think the interleaved pipeline is supported, for we use `torch.nn.Identity`. `Sequntial` is ok as well.

Reply:
I am totally wrong. It needs to be redesigned.